static/main.css

```
@@ -230,23 +254,30 @@ main.normalize {
 
 .hero h1 {
     margin: 0;
+    font-size: 3rem;
+}
+
+.hero-text {
+    margin-left: 1rem;
+    margin-right: 6rem;
+}
+
+.hero-little {
+    font-weight: 300;
+    font-size: 12px;
```

Please use a relative unit: 12px ≙ 0.75rem = 12px / 16px*

*Web browsers calculate the REM (the root em size) based on the root element size. The default for modern web browsers is 16px, so the conversion is SP_SIZE/16 = rem.

```suggestion
    font-size: 0.75rem;
```